syntax = "proto3";

message Items{
  string item_id =1;
  string product_name = 2;
  string category = 3;
  int32 quantity = 4;
  string description = 5;
  string seller_address = 6;
  float price_per_unit = 7;
  float rating = 8;
}

message ItemsResponse {
  string message = 1;
}

service NotificationService {
  rpc ReceiveNotification (Items) returns (ItemsResponse);
}
